pkg/virt-handler/vm.go

```diff
@@ -1540,7 +1546,9 @@ func isGuestAgentSupported(vmi *v1.VirtualMachineInstance, commands []v1.GuestAg
 		}
 	}
 
-	if checkSSH && !_guestAgentCommandSubsetSupported(SSHRelatedGuestAgentCommands, commands) {
+	if checkSSH &&
+		!(_guestAgentCommandSubsetSupported(SSHRelatedGuestAgentCommands, commands) ||
+			_guestAgentCommandSubsetSupported(OldSSHRelatedGuestAgentCommands, commands)) {
```

Comment:
nit - it's super easy to miss the context of the `!` here with the conditional over multiple lines, what about breaking things out to make it morer readable?

```suggestion
	if checkSSH {
		newCmdSupport := _guestAgentCommandSubsetSupported(SSHRelatedGuestAgentCommands, commands)
		oldCmdSupport := _guestAgentCommandSubsetSupported(OldSSHRelatedGuestAgentCommands, commands)
		if !(newCmdSupport || oldCmdSupport) {
			return false, "This guest agent doesn't support required public key commands"
		}
```

Reply:
done